### ISSUE TITLE
[Overall user experience] Design component or pattern in use isn't considered a best practice. (01.01.4)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#35301]
#### VFS team name
Public Websites

#### VFS product name
Outreach & Events Enhancements

#### Point of Contact/Reviewers
Sarah Koomson (@Saraharaba8) - Accessibility

---

### Platform Issue
Design component or pattern in use isn't considered a best practice.

### Issue Details
Disabling zooming is problematic for users with low vision who rely on screen magnification to properly see the contents of a web page.   [user-scalable="no"] is used in the <meta name="viewport"> element or the [maximum-scale] attribute is less than 5. The initial maximum scale was set to 1.0

### Link, screenshot or steps to recreate

![156785ED-2C05-4B7A-B812-25613888E97B](https://user-images.githubusercontent.com/95157695/149557277-44a90c79-4c1a-4570-8a43-818cf9a9c215.png)

### VA.gov Experience Standard
[Category Number 01, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
Remove the user-scalable="no" parameter from the viewport meta tag and make sure the maximum-scale parameter is set to 5 or greater 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by shiragoodman** [comment]
@Saraharaba8 could you update this ticket with the correct screen shot?

**Comment by Saraharaba8** [comment]
@shiragoodman Done. Thanks!

**Comment by srsuddath** [comment]
Hello @Saraharaba8 can you provide a url for where you saw this issue? I am inspecting the event and event detail pages and am only seeing the following meta data
![Screen Shot 2022-02-03 at 3 04 14 PM](https://user-images.githubusercontent.com/47081947/152420172-c5dd71dd-e093-4fe1-a1f3-c30d0103b6b8.png)

**Comment by Saraharaba8** [comment]
@srsuddath Hello. I am currently OOO. Will be returning and will provide link tomorrow. 

Best, 
Sarah

**Comment by srsuddath** [comment]
@Saraharaba8 can you pass me the url to this?

**Comment by Saraharaba8** [comment]
> 

Hi so sorry for the delay. I took off extra time than planned. Attached is the url https://www.cvent.com/c/calendar/a66e4100-024f-4394-b9c8-1eb784a40434. This is the calendar portion

**Comment by srsuddath** [comment]
Hey, so as far as I am aware we don't own that page, so I will be unable to make that change. We can investigate who owns that page and see about getting them to resolve the issue.

**Comment by Saraharaba8** [comment]
> Hey, so as far as I am aware we don't own that page, so I will be unable to make that change. We can investigate who owns that page and see about getting them to resolve the issue.

Thanks! @srsuddath

**Comment by shiragoodman** [comment]
hey @srsuddath  - opening this back up because we still need to ensure it's resolved, even though your team doesn't own it. Were you able to find out who owns the page? It looks like it's taking us out of VA.gov

**Comment by brianalloyd** [comment]
@shiragoodman I'm thinking CMS might help us determine ownership of the page.  I'm going to remove PW labels in support.

**Comment by shiragoodman** [comment]
@brianalloyd who from CMS do you think could help? I don't have the best understanding of how the CMS teams are organized, especially now that they've split out and created more teams. Please let me know. thanks!

**Comment by brianalloyd** [comment]
@shiragoodman good questions.  @cmaeng any idea which CMS team we should route this issue to in support?

**Comment by cmaeng** [comment]
@shiragoodman @brianalloyd the [link that was referenced](https://www.cvent.com/c/calendar/a66e4100-024f-4394-b9c8-1eb784a40434) isn't within the modernized va.gov so not sure how to proceed here. asking Jenny HL may be an option but also taking a look at the url that linked to this would be helpful too.

@Saraharaba8 how did you land on that link?

**Comment by Saraharaba8** [comment]
Hi @cmaeng. I reached this calendar link after clicking the "Prudential Pathways Seminars for US Department of Veterans Affairs - Transitioning Successfully into Retirement" tab in the "Events". section.

**Comment by Saraharaba8** [comment]
@cmaeng 
![2BF601F2-91A4-40E6-8343-6EFFFD490D4B_1_105_c](https://user-images.githubusercontent.com/95157695/154558122-72bc7d18-44aa-4660-9598-b30725cd2667.jpeg)
![8BCA768B-BA23-4F1D-BB0F-4A65C1E3AC4E](https://user-images.githubusercontent.com/95157695/154558154-a6c719fc-9fbc-4ddf-b600-fca3f60858d3.jpeg)
Then I clicked on "Link to see All Events"

**Comment by cmaeng** [comment]
thanks @Saraharaba8 .

@shiragoodman the event that was the source content that linked to the bad accessibility experience has since passed (12/15/21). is the expectation that the link be removed? or are you looking to connect with the owner of the content?

**Comment by cmaeng** [comment]
adding the source page for reference: https://www.va.gov/outreach-and-events/events/prudential-pathways-seminars-for-us-department-of-veterans-affairs-transitioning-successfully-into-retirement/

**Comment by shiragoodman** [comment]
@cmaeng I'm not sure as we've never encountered this type of problem before. Let me ask my leadership and see if they have a recommendation. I'll follow up.

**Comment by shiragoodman** [comment]
@cmaeng @brianalloyd @Saraharaba8 @srsuddath  - We will be closing this ticket as the issue is outside of VA.gov. Thank you all for helping us investigate!

**Comment by Saraharaba8** [comment]
> @cmaeng @brianalloyd @Saraharaba8 @srsuddath  - We will be closing this ticket as the issue is outside of VA.gov. Thank you all for helping us investigate!

Thanks Shira for all of your help!
